web/js/containers/sidebar/smart-handoff.js

```diff
@@ -0,0 +1,470 @@
+            text="GO TO EARTHDATA SEARCH"
+            className="red"
+            valid={isValidDownload}
+          />
+
+          { showBoundingBox && (
+            <Crop
+              className="download-extent"
+              x={x}
+              y={y}
+              width={x2 - x}
+              height={y2 - y}
+              maxHeight={screenHeight}
+              maxWidth={screenWidth}
+              onChange={this.onBoundaryChange}
+              onClose={onClose}
+              keepSelection
+              bottomLeftStyle={{
+                left: x,
+                top: y2 + 5,
+                width: x2 - x,
+              }}
+              topRightStyle={{
+                left: x,
+                top: y - 20,
+                width: x2 - x,
+              }}
+              coordinates={coordinates}
+              showCoordinates
+            />
+          )}
+        </div>
+      );
+    }
+    return (
+      <div id="smart-handoff-side-panel">
+        <h1>None of your currently listed layers are available for data download.</h1>
+      </div>
+    );
+  }
+}
+
+/**
+ * Method call to direct the user to Earthdata Search with the necessary URL parameters that
+ * encapsulate what the user is intending to try and download data / granules from
+ * @param {*} selectedDate
+ * @param {*} selectedLayer
+ * @param {*} extentCoords
+ * @param {*} showBoundingBox
+ */
+const openEarthDataSearch = (selectedDate, selectedLayer, extentCoords, showBoundingBox) => () => {
+  const { conceptId, daynight } = selectedLayer;
+  const { southWest, northEast } = extentCoords;
+
+  const startDate = `${moment.utc(selectedDate).format('YYYY-MM-DD')}T00:00:00.000Z`;
+  const endDate = `${moment.utc(selectedDate).format('YYYY-MM-DD')}T23:59:59.999Z`;
+
+  const dateRange = `${startDate},${endDate}`;
+
+  let earthDataSearchURL = `https://search.earthdata.nasa.gov/search/granules?p=${conceptId}&[qt]=${dateRange}&m=0.0!-180.0!0!1!0!0,2`;
+
+  if (daynight) earthDataSearchURL += `&pg[0][dnf]=${daynight}`;
+  if (showBoundingBox) earthDataSearchURL += `&sb=${southWest},${northEast}`;
+
+  window.open(earthDataSearchURL, '_blank');
+};
+
+/**
+ * Handle type-checking of defined properties
+ */
+SmartHandoff.propTypes = {
+  isActive: PropTypes.bool,
+  activeLayers: PropTypes.array,
+  map: PropTypes.object.isRequired,
+  boundaries: PropTypes.object,
+  proj: PropTypes.object,
+  screenHeight: PropTypes.number,
+  screenWidth: PropTypes.number,
+  selectedDate: PropTypes.instanceOf(Date),
+  showWarningModal: PropTypes.func,
+};
+
+/**
+ * ReactRedux; used for selecting the part of the data from the store
+ * that the Smarthandoff component needs. This is called every time the
+ * store state changes.
+ * @param {*} state | Encapsulates the entire Redux store state.
+ * @param {*} ownProps | Data from SmartHandoff that is used to retrieve data from the store.
+ */
+const mapStateToProps = (state, ownProps) => {
+  const { tabTypes } = ownProps;
```

Comment:
It doesn't look like you're using this prop so you can remove any references to it here, as well as remove the `ownProps` param entirely from your `mapStateToProps` definition.